### ISSUE TITLE
Training without a pre-trained model

### ISSUE THREAD

**Issue by J-H-Hwang** [rohitgirdhar/ActionVLAD#35]
I'd like to know how to train a model without a pre-trained model.
Before executing feats for clustering, I have to get a trained model, but I don't know how to get it. 
I tried to use a train_image_classifier with avg-conv pooling layer, two-layer classifier instead of a netvllad layer and linear classifier. 
I want to know how to set hyper-parameters such as learning rate and steps etc.. 

Thanks!
